### PR TITLE
fix-38380: New composer : insert youtube link

### PR DESCRIPTION
**Pull request by IlhemEssaadi**
ISSUES : added a css class named video to fix the position of the video in the post content.

### CODE REVIEW

**Review by rdenarie — CHANGES_REQUESTED** [review]
Why do you add this css rule ?

**Comment by IlhemEssaadi** [comment]
> Why do you add this css rule ?

the content of the post can contain a video that's why I added a video class to postContenet to fix the margin left of video.

**Comment by rdenarie** [comment]
When you say a "post", do you mean a "news" or an "activity" ?
In addition, my problem is that this class is used without any context. So if someone else use this class, we will have a conflict. 
"video" is a commun word, and I'm afraid of conflict.

Finally, for margin, vue provide some helper classes : https://vuetifyjs.com/en/styles/spacing/#how-it-works
So, can we use theses helpers class instead of video class ? and we should also take care of rtl-ization

**Review by rdenarie — CHANGES_REQUESTED** [review]
See my last comment

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you rebase the PR ?

**Comment by IlhemEssaadi** [comment]
> Can you rebase the PR ?

Done !!

**Review by rdenarie — APPROVED** [review]

**Review by rdenarie — CHANGES_REQUESTED** [review]
You need to take care of rtl. 
Please add the css for the margin-right also

**Comment by IlhemEssaadi** [comment]
> You need to take care of rtl. Please add the css for the margin-right also

tested , no need to add margin-right because for all rtlisation we just need to add margin-left .

**Review by rdenarie — CHANGES_REQUESTED** [review]
.

**Comment by rdenarie** [comment]
> > You need to take care of rtl. Please add the css for the margin-right also
> 
> tested , no need to add margin-right because for all rtlisation we just need to add margin-left .

how the margin-left is transformed to margin-right when you use right to left ?
Can you show me a screenshot with right-to-left ?

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less [hunk 1]
```diff
@@ -622,6 +622,11 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     ul li {
       list-style-type: disc;
     }
+    .video {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
```
Comment (rdenarie) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less:629]: No need to use flex here. Use the same behaviour as we already have with the embedded object : 
```suggestion
    .video {
      width:320px;
      margin-left:auto;
      margin-right:auto;
    }
```

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**